variables.tf

```diff
@@ -0,0 +1,37 @@
+
+
+variable "location" {
+description = "The Azure location where all resources in this deployment should be created"
+default = "uksouth"
+}
+
+variable "GITHUBID" {
+description = "The githubid"
+default = "47190936"
+}
+
+variable "WEBAPPLICATIONCLIENT" {
```

Comment:
Trying to visit your app in Azure, I get redirected to your previous version of the app (todoprojectapp vs illigeorgieva102); don't forget to update your OAuth app registration (or create a new one) so that it redirects to the correct webpage

Reply:
Updated, tahnk you